### ISSUE TITLE
Feedback week 4

### ISSUE THREAD

**Issue by lluisgomez** [XeniaSalinas/mcv-m5#2]
I've found the sections of your Overleaf article for these weeks' work well written in general. I think you can improve a bit the general context of the state of the art on object detection. e.g. instead of starting with the sentence "The main architectures for object detection ... " try to find a better reading flow between the classification and detection problems, and shortly summarize how object detection was approached before end-to-end CNNs came into the game.  You can also add a short overview of the CAM paper in the state of the art review if you show some experiments with it afterwards. 

In section 3.2 please add a bit more of implementation details: learning rate, optimizer, training from scratch or fine-tuning, etc. In section 4.2 the presentation of the results is correct but I miss a bit of critical appreciation about them: why are the results on Udacity dataset so low compared with the ones on TT100K? Similarly, I did not find any description/analysis of the datasets that helps to better understand the results. Showing some images with qualitative results may also help on this direction.

Please, fix the headings on Table 3.

On the other hand, there is some confussion when you describe the data augmentation strategy you have  followed to improve the YOLO performance. Some of the chosen values seem arbitrary, e.g. a rotation of 30 degrees seems too much for a road environment. Moreover, you mention to have used "elastic deformation" augmentation but as you can see [in the code](https://github.com/XeniaSalinas/mcv-m5/blob/master/code/tools/data_loader.py#L558) this data augmentation strategy is not implemented for detection datasets. This experiment is not clear and there should be some misunderstanding.

Regarding the slides for the presentation of the work done on these two weeks, I would recommend you to change a bit the presentation of the results on slide 13. In my opinion the bar chart with FPS values can be removed and you can show a single FPS value for each method (e.g. close to its name on the f-score chart).

Your integration of the SSD network is well done!

Although, as mentioned earlier I cannot consider task (e) to be complete with the description of the data augmentation experiment you provide, and that I miss the dataset analysis part, I acknowledge the work done on the most difficult task for these weeks (SSD integration) and the extra work done with the experiments with the CAM method. Thus, I guess your mark for these two weeks will be very close to the maximum mark.
